I1106 19:24:34.275173 22498165619584 run-alphafold-2.2.0.py:116] Mounting /home/kc5054/CHEM2671/comp-lab-class/Week7/run -> /tmp/kc5054/6bacd0c1af9b4fe6b31c38fa4503435b/fasta_path_0
Command to run: MY_ALPHAFOLD_BINDS="--bind /home/kc5054/CHEM2671/comp-lab-class/Week7/run:/tmp/kc5054/6bacd0c1af9b4fe6b31c38fa4503435b/fasta_path_0:ro" /bin/bash /scratch/work/public/apps/alphafold/2.2.0/run-alphafold-2.2.0.bash python -u /opt/alphafold/run_alphafold.py --fasta_paths=/tmp/kc5054/6bacd0c1af9b4fe6b31c38fa4503435b/fasta_path_0/rcsb_pdb_2W9T.fasta --uniref90_database_path=/alphafold-data/uniref90/uniref90.fasta --mgnify_database_path=/alphafold-data/mgnify/mgy_clusters_2018_12.fa --data_dir=/alphafold-data --template_mmcif_dir=/alphafold-data/pdb_mmcif/mmcif_files --obsolete_pdbs_path=/alphafold-data/pdb_mmcif/obsolete.dat --pdb70_database_path=/alphafold-data/pdb70/pdb70 --uniclust30_database_path=/alphafold-data/uniclust30/uniclust30_2018_08/uniclust30_2018_08 --bfd_database_path=/alphafold-data/bfd/bfd_metaclust_clu_complete_id30_c90_final_seq.sorted_opt --output_dir=/home/kc5054/CHEM2671/comp-lab-class/Week7/run --max_template_date=2020-05-14 --db_preset=full_dbs --model_preset=monomer --benchmark=False --use_precomputed_msas=False --num_multimer_predictions_per_model=5 --run_relax=True --use_gpu_relax=True --logtostderr
I1106 19:24:46.067559 22480807826624 templates.py:857] Using precomputed obsolete pdbs /alphafold-data/pdb_mmcif/obsolete.dat.
I1106 19:24:46.825214 22480807826624 xla_bridge.py:252] Unable to initialize backend 'tpu_driver': NOT_FOUND: Unable to find driver in registry given worker: 
I1106 19:24:46.949490 22480807826624 xla_bridge.py:252] Unable to initialize backend 'tpu': INVALID_ARGUMENT: TpuPlatform is not available.
I1106 19:24:54.861010 22480807826624 run_alphafold.py:376] Have 5 models: ['model_1_pred_0', 'model_2_pred_0', 'model_3_pred_0', 'model_4_pred_0', 'model_5_pred_0']
I1106 19:24:54.861226 22480807826624 run_alphafold.py:393] Using random seed 1187600187205971793 for the data pipeline
I1106 19:24:54.861421 22480807826624 run_alphafold.py:161] Predicting rcsb_pdb_2W9T
I1106 19:24:54.863998 22480807826624 jackhmmer.py:133] Launching subprocess "/usr/bin/jackhmmer -o /dev/null -A /state/partition1/job-26731406/tmpcbj1fwgl/output.sto --noali --F1 0.0005 --F2 5e-05 --F3 5e-07 --incE 0.0001 -E 0.0001 --cpu 8 -N 1 /tmp/kc5054/6bacd0c1af9b4fe6b31c38fa4503435b/fasta_path_0/rcsb_pdb_2W9T.fasta /alphafold-data/uniref90/uniref90.fasta"
I1106 19:24:55.164171 22480807826624 utils.py:36] Started Jackhmmer (uniref90.fasta) query
I1106 19:29:40.626461 22480807826624 utils.py:40] Finished Jackhmmer (uniref90.fasta) query in 285.462 seconds
I1106 19:29:40.885035 22480807826624 jackhmmer.py:133] Launching subprocess "/usr/bin/jackhmmer -o /dev/null -A /state/partition1/job-26731406/tmp2u8_14lu/output.sto --noali --F1 0.0005 --F2 5e-05 --F3 5e-07 --incE 0.0001 -E 0.0001 --cpu 8 -N 1 /tmp/kc5054/6bacd0c1af9b4fe6b31c38fa4503435b/fasta_path_0/rcsb_pdb_2W9T.fasta /alphafold-data/mgnify/mgy_clusters_2018_12.fa"
I1106 19:29:41.095702 22480807826624 utils.py:36] Started Jackhmmer (mgy_clusters_2018_12.fa) query
slurmstepd: error: *** JOB 26731406 ON gr033 CANCELLED AT 2022-11-06T19:29:55 ***
